### PR TITLE
Requesting signoff for Gas Subsystem

### PR DESCRIPTION
**Pull request by Lmcarson20**

### CODE REVIEW

File: Documentation/Signoffs/lukecarson-gas-signoff.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Gas Module Signoff
+
+## Subsystem Function
+The gas module plays a critical role in ensuring safety and well-being by detecting three gases that can be found in homes: propane, carbon monoxide, and ammonia. The module collects data on gas concentrations and wirelessly transmits it to the head unit through our wireless communication module. At the head unit, the data undergoes analysis to ensure compliance with OSHA standards for an eight-hour monitoring window. This process provides real-time data and alerts about gas concentrations, helping the system respond to and manage safety or environmental concerns. The module is a part of our overall system, seamlessly working to preserve the environment of the house and comply with regulatory standards. The module’s performance is essential to maintaining a safe and healthy living environment. Any malfunction of the gas sensor module could result in potential risks, such as gas leaks or air quality degradation. Therefore, its function is crucial in achieving the system's primary goal of providing accurate, timely, and actionable gas-related information while adhering to established safety standards and guidelines.
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/lukecarson-gas-signoff.md]: Why is this a single paragraph?

**Review by JesseTNRoberts — COMMENTED** [review]
How do these communicate with the micro? Also, what resolution is required?

**Comment by Lmcarson20** [comment]
> How do these communicate with the micro? Also, what resolution is required?

This is addressed in the first subscript of the analysis section of the updated signoff

File: Documentation/Signoffs/lukecarson-gas-signoff.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+# Gas Module Signoff
+
+## Subsystem Function
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/lukecarson-gas-signoff.md:3]: Be brief and to the point regarding what this is supposed to do.

**Review by JesseTNRoberts — COMMENTED** [review]
You need to identify the resolution of your signal at the microcontroller. How small of a step size in ppm do you need to be able to detect? Can you?

File: Documentation/Signoffs/lukecarson-gas-signoff.md (hunk 2 above)
Comment (Lmcarson20) [reply Documentation/Signoffs/lukecarson-gas-signoff.md:3]: Fixed the first paragraph where it is brief and to the point.

**Comment by Lmcarson20** [comment]
To address the smallest ppm step size that the gas module needs to detect is under the first subscript-second paragraph in the analysis section

**Review by JesseTNRoberts — COMMENTED** [review]
Paragraph format for analysis like the analog values is not appropriate. use in line equations with comments to go through the calculations.

**Comment by Lmcarson20** [comment]
I have updated the paragraph analysis to use in-line equations and comments above each solution.

**Review by JesseTNRoberts — COMMENTED** [review]
I thought the analog input was 0-3.3

How much resolution do we need? (you have described the resolution we have, but I'm not clear on if this is enough because there is no appropriate constraint)

How does noise affect the readings? How much noise is acceptable?

**Comment by Lmcarson20** [comment]
As discussed in class, there will be a voltage divider to bring the voltage down to 3.3V so the data will be in range for the ADC on esp.

Added constraint 3 to show that the resolution will be sufficient.

Added constraint 11, to model the noise of the wall power and the affects it has on our system.

**Review by JesseTNRoberts — COMMENTED** [review]
Ok, there's 12 mV. So, what? Is that good or bad and based on what?

**Comment by Lmcarson20** [comment]
Revision and further explanation of noise simulation

**Review by JesseTNRoberts — COMMENTED** [review]
add filter to schematic.

**Comment by Lmcarson20** [comment]
Added filter to schematic and updated BOM.

**Review by JesseTNRoberts — APPROVED** [review]

**Pull request merged**